lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -30,17 +31,21 @@
     cursor: pointer;
 
     .nx-radio-checkbox__control {
-      border-color: var(--nx-color-interactive-border-hover);
-      stroke: var(--nx-color-interactive-border-hover);
+      // We are using box-shadow below instead of border
+      // so we do not have to re-calculate the focus Pseudo-element in NxRadio and NxCheckbox
+      // when the width changes in dark mode.
+      box-shadow: inset 0 0 0 var(--nx-radio-checkbox-border-width) var(--nx-color-interactive-border-hover);
```

Comment:
You previously had a mixin for these box-shadows.  That seemed valuable due to the repetition; why'd you get rid of it?

Reply:
> > Brought it back here: [d6bab91](https://github.com/sonatype/sonatype-react-shared-components/commit/d6bab913222219f39f54653c5cf49057ff9def54)
> 
> Rather than duplicating it in each file, I'd say it should be in a helper file. The one for the more complicated disabled box-shadow should probably move to that same file

I moved it to its own file here: 66373d319bea3d6929c75b8ff76835f5cc99574a
Although, I feel that it might be simpler before (less coupling), not sure.
Let me know what you think!

Reply:
> > I moved it to its own file here: [66373d3](https://github.com/sonatype/sonatype-react-shared-components/commit/66373d319bea3d6929c75b8ff76835f5cc99574a)
> > Although, I feel that it might be simpler before (less coupling), not sure.
> > Let me know what you think!
> 
> You missed the one in `_nx-radio-checkbox.scss`, which was a big part of my point as it's identical to one of the others.

Ah right!
Fixed:
02fc06491d76cc47ea2a483a6238180066df55b6
ca6524f7561dd171d85894971055cd491513c9e1